src/main/java/io/anserini/index/Posting.java

```diff
@@ -0,0 +1,54 @@
+/**
```

Comment:
I would make this a static inner class of `IndexReaderUtils` since it's not supposed to have independent existence. We don't want to cause confusion with native Lucene classes.